pom.xml

```diff
@@ -63,23 +63,24 @@
 
   <modules>
     <module>common-kafka-test</module>
-    <module>common-kafka-admin</module>
     <module>common-kafka</module>
     <module>common-kafka-connect</module>
   </modules>
 
   <properties>
     <hamcrest.version>1.3</hamcrest.version>
     <jackson.version>2.12.6.1</jackson.version>
-    <kafka.version>2.3.1</kafka.version>
-    <scala.major.version>2.12</scala.major.version>
-    <scala.version>${scala.major.version}.8</scala.version>
+    <kafka.version>3.2.0</kafka.version>
```

Comment:
Unless there is a specific technical reason for 3.2.0, let's bump this up to 3.4.0 which was just [released](https://github.com/apache/kafka/releases/tag/3.4.0) today.

Reply:
We are targeting 3.2.0 because that is the version used on some internal clusters so we wanted to avoid any version conflicts on the classpath.

Reply:
Ok